src/Libraries/DSCPython/DSCPython.csproj

```diff
@@ -25,6 +25,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
+    <LangVersion>7.1</LangVersion>
```

Comment:
I had to update the lang version here to 7.1 to get the DSCPython project build. I was getting this error: https://github.com/DynamoDS/Dynamo/pull/10830#issuecomment-652030087. @mmisol How did you get around it?

Reply:
I was using 2017 VS. Will move to 2019 and remove this change.